app/models/fraud_event.rb

```diff
@@ -0,0 +1,10 @@
+# Feel free to propose a better name.
+# We need to stash the user's irs_session_id when a user is flagged for fraud.
+# The IRS asked us to store our session ID, too.
+# We give this back to them when a user is manually approved/rejected.
+# It COULD make sense to put other stuff here and make this the primary way of tracking this,
+# but the immediate need is to just have a way to link this data back to a user before there
+# is a ServiceProviderIdentity.
+class FraudEvent < ApplicationRecord
```

Comment:
FWIW I'm kind of using this as a placeholder name. I worry "event" is overloaded in meaning.

The problem is that _really_ the class name should be something like WeirdTableToStashSessionIdsInThisEdgeCase, and I haven't been able to find a clear name for it.

Reply:
maybe `FraudReview`? like the user is being reviewed for possible fraud, it's not confirmed fraud (which is what `FraudEvent` implies to me)

Reply:
This is definitely an improvement. That said, I think of "fraud review" as when happens when someone goes through these to approve/reject. But I like FraudReview a lot more.

Reply:
`FraudReviewRequest`? something like that